export.py

```diff
@@ -89,200 +89,203 @@ def export_formats():
     return pd.DataFrame(x, columns=['Format', 'Argument', 'Suffix', 'CPU', 'GPU'])
 
 
+def export_decorator(fcn):
+    # YOLOv5 export decorator context manager
```

Comment:
This `fcn` abbreviation can be confused with full convolutional network, maybe `func` is much better?

Reply:
great point!

Reply:
@zhiqwang good point. I've renamed and clarified the arguments, and updated fcn to func in another function I found in general.py.